src/main/webapp/i18n/en/aboutUs.json

```diff
@@ -56,7 +56,7 @@
             "text": "<b>High user satisfaction:</b> Artemis is easy to use and provides guided tutorials. Developers focus on usability, user experience, and performance."
         },
         "customizable": {
-            "text": "<b>Customizable:</b> It supports multiple instructors, editors, and tutors per course and allows instructors to customize many course settings."
+            "text": "<a href={{ customizableUrl }}>Customizable:</a> It supports multiple instructors, editors, and tutors per course and allows instructors to customize many course settings."
```

Comment:
"text": "<a href={{ customizableUrl }}>Customizable: It supports multiple Instructors, Editors, and Tutors per course and allows Instructors to customize many course settings."